[2/3] OSDOCS#16925: Apply CQA for etcd (Backup and restore) (#116890)
openshift/openshift-docs: etcd/etcd-backup-restore/etcd-disaster-recovery.adoc

Reviewer: I'd suggest the following changes:
- Move the content from lines 21 - 24 out of this assembly and into `modules/dr-restoring-etcd-quorum-ha.adoc`
- Move the note from line 35 out of this assembly and into the `modules/dr-restoring-etcd-quorum-ha.adoc` file
- After the include statement to `modules/dr-restoring-etcd-quorum-ha.adoc`, add the "Additional resources" from lines 28 - 31 
- In that "Additional resources" section, be sure to add an xref to "Replacing an unhealthy member" (`xref:../../backup_and_restore/control_plane_backup_and_restore/replacing-unhealthy-etcd-member.adoc#replacing-unhealthy-etcd-member`), which was originally in the note on line 35.

:_mod-docs-content-type: ASSEMBLY
[id="etcd-disaster-recovery"]
= Disaster recovery
include::_attributes/common-attributes.adoc[]
:context: etcd-disaster-recovery

toc::[]

[role="_abstract"]

> To return your cluster to a working state after quorum loss, control plane failure, or expired certificates, follow the disaster recovery procedures that apply to your situation. You can restore etcd quorum, restore the cluster from an etcd snapshot, or recover from expired control plane certificates.

Reviewer: When this content moves to DITA, it will be a hard requirement for short descriptions to be 300 characters or fewer. Unfortunately, the vale checks don't look for the character limit, so it's easy to miss. Here's a suggested edit:

```suggestion
To return your cluster to a working state after quorum loss, control plane failure, or expired certificates, follow the disaster recovery procedures for your situation. You can restore etcd quorum, restore the cluster from an etcd snapshot, or recover from expired control plane certificates.
```

You can use the `quorum-restore.sh` script to restore etcd quorum on clusters that are offline due to quorum loss. When quorum is lost, the {product-title} API becomes read-only. After quorum is restored, the {product-title} API returns to read/write mode.

[NOTE]
====
If you have a majority of your control plane nodes still available and have an etcd quorum, replace a single unhealthy etcd member.
====

> 

Reviewer: I'd suggest moving the content from lines 31 - 43 out of this assembly and into `modules/dr-restoring-cluster-state-about.adoc`.

Before performing a restore, see "About restoring to a previous cluster state" for more information on the impact to the cluster.
====

// Restoring etcd quorum for high availability clusters

> include::modules/dr-restoring-etcd-quorum-ha.adoc[leveloffset=+2]

Reviewer: See my previous comment about moving the content from this assembly to this module. After you do that, you can place the Additional resources after this include statement, and move the include statement from being `leveloffset=+2` to `leveloffset=+1`.

// Restoring to a previous cluster state
include::modules/dr-restoring-cluster-state.adoc[leveloffset=+2]

> [role="_additional-resources"]

Reviewer: This "Additional resources" section was fine as-is and doesn't need to be removed.

> [role="_additional-resources"]

Reviewer: If you make the changes I suggested earlier in this file, this "Additional resources" section only needs to include an xref to "Restoring to a previous cluster state".
